.changeset/cyan-ears-wonder.md

```diff
@@ -0,0 +1,50 @@
+---
+'merchant-center-application-template-starter': minor
+'@commercetools-frontend/mc-scripts': minor
+'playground': minor
+'@commercetools-website/custom-applications': minor
+---
+
+New CLI options! 🎉
+
+## Loading dotenv files
```

Comment:
Maybe more info on this [here](https://tenor.com/view/clap-applause-proud-wow-amazed-gif-15751919)

Congrats on 10 years Nicola!! :tada:
now to the next Goal!